recipes/torchvision-extra-decoders/0002-Allow-to-specify-git-sha-through-environment-variabl.patch

```diff
@@ -0,0 +1,43 @@
+From 70c6a86d4023088a665d3b0d28344e383b4e355b Mon Sep 17 00:00:00 2001
+From: Mark Harfouche <mark.harfouche@gmail.com>
+Date: Sun, 16 Feb 2025 02:04:10 +0100
+Subject: [PATCH 2/2] Allow to specify git sha through environment variable
+
+---
+ setup.py | 20 ++++++++++----------
+ 1 file changed, 10 insertions(+), 10 deletions(-)
+
+diff --git a/setup.py b/setup.py
+index e2402de..8def746 100644
+--- a/setup.py
++++ b/setup.py
+@@ -19,16 +19,16 @@ ROOT_DIR = Path(__file__).absolute().parent
+ def get_and_write_version():
+     with open(ROOT_DIR / "version.txt") as f:
+         version = f.readline().strip()
+-    sha = "Unknown"
+-
+-    try:
+-        sha = (
+-            subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=str(ROOT_DIR))
+-            .decode("ascii")
+-            .strip()
+-        )
+-    except Exception:
+-        pass
++    sha = os.getenv("BUILD_GIT_SHA", "Unknown")
++
++    # try:
++    #     sha = (
++    #         subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=str(ROOT_DIR))
++    #         .decode("ascii")
++    #         .strip()
++    #     )
++    # except Exception:
++    #     pass
```

Comment:
I think it's better to remove old bits entirely in patches, rather than comment them out. They're still present in context, and this just doubles the patch size.

Reply:
Thanks. With the "additional cleanups" i might have increase the size of the patches, but I think it should work now!